Pod/Classes/PINRemoteImageManager.h

```diff
@@ -424,6 +424,17 @@ typedef void(^PINRemoteImageManagerProgressDownload)(int64_t completedBytes, int
                              progressImage:(nullable PINRemoteImageManagerImageCompletion)progressImage
                                 completion:(nullable PINRemoteImageManagerImageCompletion)completion;
 
+
+/**
+ Adds an image manually into the memory and disk cache.
+ 
+ @param data NSData with the raw image data.
+ @param url NSURL where the image resides.
+ @processorKey NSString key to uniquely identify processor and process. Will be used for caching processed images.
+ @additionalCost NSUInteger The cache cost of the object.
+ */
+- (BOOL) insertImageDataIntoCache:(nonnull NSData*)data withURL:(nonnull NSURL *)url processorKey:(nullable NSString *)processorKey additionalCost: (NSUInteger) additionalCost;
```

Comment:
@garrettmoon Based on how it is ultimately used here: https://github.com/pinterest/PINRemoteImage/blob/master/Pod/Classes/PINRemoteImageManager.m#L1432, `additionalCost` or `additionalCacheCost` would be most accurate. I think `additionalCacheCost` is best, as I was initially confused on what it does.

No problem on removing that space 😃